core/templates/domain/story_viewer/ReadOnlyStoryNodeObjectFactory.ts

```diff
@@ -51,12 +53,15 @@ export class ReadOnlyStoryNode {
   _explorationId: string;
   _explorationSummary: ExplorationSummary;
   _completed: boolean;
+  _thumbnailBgColor: string;
+  _thumbnailFilename: string;
 
   constructor(id: string, title: string, description: string,
       destinationNodeIds: Array<string>, prerequisiteSkillIds: Array<string>,
       acquiredSkillIds: Array<string>, outline: string,
       outlineIsFinalized: boolean, explorationId: string,
-      explorationSummary: ExplorationSummary, completed: boolean) {
+      explorationSummary: ExplorationSummary, completed: boolean,
+      thumbnailBgColor: string, thumbnailFilename: strin) {
```

Comment:
```suggestion
      thumbnailBgColor: string, thumbnailFilename: string) {
```

typescript compilation didn't complain?

Reply:
Oh, I didn't run typescript tests, will fix it.

Reply:
Done